kubernetes/configMaps/filterlist.txt

```diff
@@ -0,0 +1,24 @@
+resource://
+chromenull://
+chrome-extension://
+safari-extension://
+mxjscall://
+webviewprogressproxy://
+res://
+mx://
+safari-resource://
+chromeinvoke://
+chromeinvokeimmediate://
+mbinit://
+opera://
+localhost
+127.0.0.1
+none://
+about:blank
+android-webview
+ms-browser-extension
+wvjbscheme://__wvjb_queue_message__
+nativebaiduhd://adblock
+bdvideo://error
+https://fonts.gstatic.com/s/roboto/v18/KFO
+https://fonts.gstatic.com/s/googlesans/v9/4Ua
```

Comment:
These look like recent additions?

Reply:
Fonts urls removed. Now `./sample.filterlist.txt` and `kubernetes/configMaps/filterlist.txt` match